internal/app/connectconformance/testsuites/cancellation.yaml

```diff
@@ -32,11 +32,11 @@ testCases:
     method: ClientStream
     streamType: STREAM_TYPE_CLIENT_STREAM
     cancel:
-      afterCloseSendMs: 50
+      afterCloseSendMs: 5
     requestMessages:
     - "@type": type.googleapis.com/connectrpc.conformance.v1.ClientStreamRequest
       responseDefinition:
-        responseDelayMs: 100
```

Comment:
This seemed a bit tight -- this gives a margin for error of only 50 milliseconds, which seems like it could occasionally be exceeded on slow and/or highly-loaded CI worker machines. So I expanded it to 195 millis.